client/my-sites/marketing/index.js

```diff
@@ -17,7 +16,6 @@ import {
 	redirectConnections,
 	redirectMarketingTools,
 	redirectSharingButtons,
-	redirectTraffic,
```

Comment:
We should remove the [corresponding function](https://github.com/Automattic/wp-calypso/blob/5c45760e174642b632c69ccf2302fb777e1f75ab/client/my-sites/marketing/controller.js#L41) as well.